rasa_core/utils.py

```diff
@@ -293,7 +293,8 @@ def construct_yaml_str(self, node):
     if sys.maxunicode == 0xffff:
         yaml.reader.Reader.NON_PRINTABLE = re.compile(
                 '[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\ud83d\uE000-\uFFFD'
-                '\ude00-\ude50]')
+                '\ude00-\ude50-\udc4d\ud83c\udf89\ude80\udc4c\ud83e\uddde'
```

Comment:
this is a bit of an odd syntax - usually the pattern is: `[A-Za-z]` but now there are two dashes: `\ude00-\ude50-\udc4d` does that work?

Reply:
haha that was unintentional, idk how that worked tbh. i'll get rid of the second dash